### PR TITLE
Move flags in config.go into main.go

### LINKED ISSUE #2671: Move flags in config.go into main.go
**What would you like to be added**:
Move flags in config.go into main.go where config.New() is called

**Why is this needed**:
Flags are variables in main.go

### PR DESCRIPTION
**Pull request by Liuhaai**
fix #2671

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -34,18 +34,10 @@ import (
 // the default value in Default var.
 
 func init() {
-	flag.StringVar(&_overwritePath, "config-path", "", "Config path")
-	flag.StringVar(&_secretPath, "secret-path", "", "Secret path")
-	flag.StringVar(&_subChainPath, "sub-config-path", "", "Sub chain Config path")
```
Comment (CoderZhi) [inline config/config.go:39]: _plugins should be removed too

File: config/config.go [hunk 2]
```diff
@@ -431,7 +423,7 @@ type (
 // New creates a config instance. It first loads the default configs. If the config path is not empty, it will read from
 // the file and override the default configs. By default, it will apply all validation functions. To bypass validation,
 // use DoNotValidate instead.
-func New(validates ...Validate) (Config, error) {
+func New(_overwritePath string, _secretPath string, validates ...Validate) (Config, error) {
```
Comment (CoderZhi) [inline config/config.go:426]: merge _overwritePath and _secretPath into `configPaths []string`. in line 430, loop through these paths:
```
    for _, path in range configPaths {
        if path != "" {
            opts = append(opts, uconfig.File(path))
        }
    }
```

File: config/config.go [hunk 3]
```diff
@@ -479,7 +471,7 @@ func New(validates ...Validate) (Config, error) {
 }
 
 // NewSub create config for sub chain.
-func NewSub(validates ...Validate) (Config, error) {
+func NewSub(_secretPath string, _subChainPath string, validates ...Validate) (Config, error) {
```
Comment (CoderZhi) [inline config/config.go:474]: same here

**Review by CoderZhi — COMMENTED** [review]
good job, it is ready to review

**Review by CoderZhi — APPROVED** [review]

**Review by raullenchai — APPROVED** [review]

**Pull request merged**